azure-pipelines.yml

```diff
@@ -42,6 +42,8 @@ stages:
             python.version: '3.9'
           Python310:
             python.version: '3.10'
+          Python311:
+            python.version: '3.11'
```

Comment:
What is this weird indent?

Reply:
That's how it goes :) The indentation is automatically checked btw.

Reply:
Ahah sorry, my phone where showing really weird things ;)